packages/common/src/store/upload/types.ts

```diff
@@ -57,11 +57,15 @@ export interface TrackMetadataForUpload extends Omit<TrackMetadata, 'artwork'> {
  * Unlike normal CollectionMetadata, CollectionMetadataForUpload has artwork
  * and track details to be passed to its descendant tracks.
  */
-export interface CollectionMetadataForUpload extends CollectionMetadata {
-  artwork: {
-    file?: Blob
-    url: string
-  }
+export interface CollectionMetadataForUpload
```

Comment:
Should probably make sure this doesn't mess with @sliptype 's work for uploading via SDK.
I think a pattern of using a separate property to indicate a _file_ to be uploaded, resulting in a CID that can be added to the metadata might make more sense than overloading the `artwork` field and requiring runtime checks? Understandable though if there's deep legacy logic in the cache that's also doing this.
For example: https://github.com/AudiusProject/audius-protocol/blob/db3f11a9a7e9615a0ba36f3ddd5f4f73bfe27d5b/packages/sdk/src/sdk/api/users/types.ts#L10